carball/analysis/analysis_manager.py

```diff
@@ -177,20 +231,38 @@ def write_pandas_out_to_file(self, file):
     def get_protobuf_data(self) -> game_pb2.Game:
         """
         :return: The protobuf data created by the analysis
+
+        USAGE: A Protocol Buffer contains in-game metadata (e.g. events, stats)
+
+        INFO: The Protocol Buffer is a collection of data organized in a format similar to json. All relevant .proto
+        files found at https://github.com/SaltieRL/carball/tree/master/api.
+
+        Google's developer guide to protocol buffers may be found at https://developers.google.com/protocol-buffers/docs/overview
         """
         return self.protobuf_game
 
     def get_data_frame(self) -> pd.DataFrame:
+        """
+        :return: The pandas.DataFrame object.
+
+        USAGE: A DataFrame contains in-game frame-by-frame data.
+
+        INFO: The DataFrame is a collection of data organized in a format similar to csv. The 'index' column of the
+        DataFrame is the consecutive in-game frames, and all other column headings (150+) are tuples in the following
+        format:
+            (Object, Data), where the Object is either a player, the ball or the game.
+
+        All column information (and keys) may be seen by calling data_frame.info(verbose=True)
```

Comment:
We might want to copy and paste that into a wiki or maybe look kinda a GitHub action that creates this information at every push to master

Reply:
https://github.com/marketplace/actions/wiki-page-creator-action